src/datasources/results/components/query-builders/query-steps/StepsQueryBuilder.tsx

```diff
@@ -101,7 +102,7 @@ export const StepsQueryBuilder: React.FC<StepsQueryBuilderProps> = ({
         ...stepsPathField.lookup,
         dataSource: [
           ...(stepsPathField.lookup?.dataSource || []),
-          ...stepsPath.map(path => ({ label: path, value: path })),
+          ...stepsPath.map(path => ({ label: path.label, value: path.value })),
```

Comment:
```suggestion
          ...stepsPath
```

Since the types are same, can't we just do this instead of `Array.map()`? This would mean there is one less copy of the `stepPath`